### DISCUSSION TITLE
Missing config files??

### DISCUSSION THREAD

**Discussion by dlvommen76** [thijsvanloef/palworld-server-docker#247] [Q&A]
Is anyone else having issues locating their PalWorldSettings.ini file? I know for a fact where I started this container, I know the directory, when I go to that directory its empty. I stopped the docker container to check, still nothing.

**Comment by thijsvanloef** [comment]
Hi @dlvommen76 Please make sure that you are running the latest docker container:
`docker compose down --rmi all`
`docker compose up -d `

And try to look for the files in the volume you have mounted your container on:
```yaml
      volumes:
         - ./palworld:/palworld/
```
